sync_test.go

```diff
@@ -34,6 +35,23 @@ func TestFsSyncer_Sync(t *testing.T) {
 		"it should copy a directory": {
 			fixtureSrc: "src/dir",
 		},
+		"it should copy a hard link": {
```

Comment:
In this test we want to test that the files `a` and `b` are hard links. But this test seems impossible to do on the CI as hard links are not supported in Git: https://stackoverflow.com/a/3731139/1351436